front/src/modules/ui/modal/components/types/ModalTypes.tsx

```diff
@@ -0,0 +1,2 @@
+export type ModalSize = 'small' | 'medium' | 'large';
```

Comment:
our current convention is to only have one type per file

Reply:
or rather keep them in Modal.tsx

Reply:
moved them to Modal.tsx